src/modules/job-info/list.c

```diff
@@ -491,7 +491,7 @@ void list_id_cb (flux_t *h, flux_msg_handler_t *mh,
 void list_attrs_cb (flux_t *h, flux_msg_handler_t *mh,
                     const flux_msg_t *msg, void *arg)
 {
-    if (flux_respond_pack (h, msg, "{s:[s,s,s,s,s,s,s,s,s,s,s,s,s]}",
+    if (flux_respond_pack (h, msg, "{s:[s,s,s,s,s,s,s,s,s,s,s,s,s,s]}",
```

Comment:
Up to you, but would be less error prone to either return a constant JSON string here, or have a function that builds the array from a constant array of strings by iteratively calling `json_array_append()`?

Reply:
yeah, i've been lazy waiting until I hit 80 chars :-)  I'll add this to the cleanup list for a follow up PR.